cluster/src/main/scala/kanaloa/reactive/dispatcher/ClusterAwareBackend.scala

```diff
@@ -44,13 +44,22 @@ class ClusterAwareBackend(
   override def apply(f: ActorRefFactory): Future[ActorRef] = {
     val retriever = f.actorOf(ClusterAwareBackend.retrieverProps(router, routingLogic))
     import system.dispatcher
-    (retriever ? ClusterAwareBackend.GetRoutee).mapTo[RouteeRef].map(_.actorRef)
+    retriever.ask(ClusterAwareBackend.GetRoutee)(timeout = timeout.duration * 2).mapTo[RouteeRef].map(_.actorRef) //let the retriever handles the time internally.
```

Comment:
the `retriever` is going to handle timeout internally but the ask requires a timeout, so just make it longer than the timeout passed into `retriever`.